src/main/java/digital/slovensko/autogram/ui/gui/SettingsDialogController.java

```diff
@@ -259,8 +266,9 @@ private void initializeCustomKeystoreSettings() {
         });
     }
 
-    public void onCancelButtonAction() {
-        var stage = (Stage) closeButton.getScene().getWindow();
+    public void onSaveButtonAction() {
+        userSettings.save();
```

Comment:
Tu som zmenil spravanie. Imho je to takto lepsie, explicitnejsie a vies zmenit viac veci naraz. Je na zvazenie co tam nedat button co to neulozi. cc @celuchmarek